.gitignore

```diff
@@ -1,2 +1,3 @@
-/venv
-/.idea
\ No newline at end of file
+venv/
+.idea/
+output_configs/
```

Comment:
what does diviete icon means? 
what is /.idea?
is outp_configs your folder to save results?

Reply:
In .gitignore '/' at the end of a word indicates that it is a directory to ignore;
.idea/ is a directory PyCharm creates with all the information about the project. I added it here to avoid committing useless files, but if you don't like it I can remove it and decide locally what to ignore; 
Yes, output_config is a folder to put the output. I put it here so we avoid committing the output of the script